app/views/servers/show.html.erb

```diff
@@ -214,7 +214,9 @@
               <%= Server.human_attribute_name(:photo) %>
             <% end %>
 
-            <%= image_tag @server.photo.variant(geometry: "x220"), class: "server_photo ms-0 mb-3" %>
+            <%= link_to @server.photo, rel: :noopener, target: :_blank do %>
+              <%= image_tag @server.photo, class: "w-100" %>
```

Comment:
keep variant use. Could be a bigger one than `x220`. But may be nice to still have one.